src/com/facebook/buck/swift/SwiftDescriptions.java

```diff
@@ -54,6 +60,16 @@ private SwiftDescriptions() {
       BuildTarget buildTarget) {
     output.srcs = args.srcs.transform(
         input -> filterSwiftSources(sourcePathResolver, args.srcs.get()));
+    output.exportedHeaders = args.exportedHeaders.transform(
```

Comment:
We pass a flag to generate it, so yes: https://github.com/facebook/buck/pull/911/files#diff-6d7c1429a28d3416822c55287d317a40R190